src/main/java/Command/EchoCommand.java

```diff
@@ -0,0 +1,50 @@
+package command;
+import duke.DukeException;
+import storage.Storage;
+import taskList.TaskList;
+import ui.Ui;
+
+/**
+ * The `EchoCommand` class represents a command to display an echo message to the user in the Duke application.
+ * It extends the `Command` class and is responsible for displaying a user-specified message as an echo response.
+ * This class encapsulates the behavior of echoing a message to the user.
+ *
+ * @author raydenlim
+ * @version 0.0.0
+ */
+public class EchoCommand extends Command {
```

Comment:
Is this class still needed? Since our chatbot no longer echos commands

Reply:
secret function ;)